apps/services/form-system/src/app/modules/listItems/models/dto/updateListItem.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { LanguageType } from '../../../../dataTypes/languageType.model'
+
+export class UpdateListItemDto {
+  @ApiProperty()
+  label!: LanguageType
+
+  @ApiProperty()
```

Comment:
You probably need `@ApiProperty{ type: LanguageType })` when it is not a base type and in this case also `@ApiPropertyOptional`